### PR TITLE
Propagate summary tone of BIP Comments to their applicable BIP preambles

### PR DESCRIPTION
**Pull request by luke-jr**
Affects: BIPs 38, 39, 42, 47, 60, 61, 74, 75, 90, 150, 151, 152

Will merge after a week or so. Anyone who disagrees with the assessments may feel free to post on the Comments wiki and/or this PR by then.

For reference, BIP 2 specifies how to post comments: https://github.com/bitcoin/bips/blob/master/bip-0002.mediawiki#BIP_comments

### CODE REVIEW

**Comment by luke-jr** [comment]
For reference, BIP 2 specifies how to post comments: https://github.com/bitcoin/bips/blob/master/bip-0002.mediawiki#BIP_comments

**Comment by jonasschnelli** [comment]
ACK for 150, 151

**Comment by jnewbery** [comment]
NACK. I don't think this is helpful information.

**Comment by btcdrak** [comment]
@jnewbery The process is already detailed by BIP2.

File: bip-0038.mediawiki [hunk 1]
```diff
@@ -4,7 +4,7 @@
   Title: Passphrase-protected private key
   Author: Mike Caldwell <mcaldwell@swipeclock.com>
           Aaron Voisine <voisine@gmail.com>
-  Comments-Summary: No comments yet.
```
Comment (Priroch1979) [inline bip-0038.mediawiki:7]: Comments-Summary: Unanimously Discourage for implementation
 
 
Leave a comment
ไม่ได้เลือกไฟล์ใด
Attach files by dragging & dropping, selecting or pasting them.

**Pull request merged**